.github/workflows/jira-issue.yaml

```diff
@@ -0,0 +1,47 @@
+name: "Mirror new issue to Jira for grooming"
+
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  create-issue:
+    runs-on: ubuntu-latest
+
+  env:
+    JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
+
+  steps:
+    - name: Installs Jira CLI
+      uses: atlassian/gajira-cli@master
+      with:
+        version: 1.0.23
+
+    - name: Create issue
+      run: |
+        jira create \
+        --noedit \
+        -e ${{ secrets.JIRA_BASE_URL }} \
+        -u ${{ secrets.JIRA_USER_EMAIL }} \
+        -p ATL -i Task \
```

Comment:
Maybe it's better to place the project name in the secrets?

Reply:
Yeah, done.